string request_path
string new_value_dict
---
bool response_code